### PR TITLE
Add some CNCF branding.

### PR DESCRIPTION
**Pull request by juliusv**
**Landing page open-source section:**
cncf_os

**Community section**
cncf_community

**Overview section**
cncf_overview

### CODE REVIEW

**Comment by juliusv** [comment]
The logo looked awkward when including it on the Overview page as well, so I omitted it there.

**Comment by juliusv** [comment]
FYI: Did this after chatting with Chris A. about adding CNCF branding to our page, which I think is a good idea.

File: content/docs/introduction/overview.md [hunk 1]
```diff
@@ -12,7 +12,10 @@ monitoring and alerting toolkit originally built at
 [SoundCloud](http://soundcloud.com). Since its inception in 2012, many
 companies and organizations have adopted Prometheus, and the project has a very
 active developer and user [community](/community). It is now a standalone open source project
-and maintained independently of any company.
+and maintained independently of any company. To emphasize this and clarify
+the project's governance structure, Prometheus joined the
+[Cloud Native Computing Foundation](https://cncf.io/) in 2016
+as a second hosted project after [Kubernetes](http://kubernetes.io/).
```
Comment (grobie) [inline content/docs/introduction/overview.md]: s/as a/as the/? There is only one second project.

**Comment by grobie** [comment]
Cool! 👍

File: content/docs/introduction/overview.md (hunk 1 above)
Comment (juliusv) [inline content/docs/introduction/overview.md]: Changed.

**Comment by fabxc** [comment]
👍

**Comment by brian-brazil** [comment]
:+1:

**Comment by caniszczyk** [comment]
Thanks :)

**Comment by juliusv** [comment]
Forgot to add the logo image itself to git. Pushed a fixup commit to master. Should be online in a sec.

**Comment by fabxc** [comment]
@juliusv you didn't check in the logo.

**Comment by juliusv** [comment]
@fabxc see my comment above :)

**Comment by caniszczyk** [comment]
Thanks! I also made a small request in the README too:
https://github.com/prometheus/prometheus/pull/1850

**Pull request merged**